src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import Page from './Page';
+
+const App = () => {
+  const [count, setCount] = useState(0);
+
+  const handleClick = () => {
+    setCount(count + 1);
+  };
+
+  const handleClickNumber = (number) => {
+    setCount(count + number);
+  };
+
+  return (
+    <Page counter={count} onClick={handleClick} onClickNumber={handleClickNumber} />
```

Comment:
```suggestion
    <Page 
      counter={count}
      onClick={handleClick}
      onClickNumber={handleClickNumber}
     />
```

저는 `props` 가 많은 경우(개인적인 기준은 2개이상) 다음과 같이 줄바꿈을 해서 코드를 정리해주는 편인데
코드 가독성을 비교했을 때 어떤게 더 나은 것 같나요?

Reply:
개인적으로 트레이너님이 하신게 좀 더 가독성에 좋다고 생각됩니다. 피드백 감사합니다.